apps/mobile/src/libs/components/input/styles.ts

```diff
@@ -1,8 +1,11 @@
 import { StyleSheet } from "react-native";
 
+import { GradientColor } from "~/libs/enums/enums";
+
+const colorIndex = 0;
 const styles = StyleSheet.create({
 	input: {
-		borderColor: "gray",
+		borderColor: GradientColor.BLUE[colorIndex],
```

Comment:
In this case we can do just like this
```suggestion
		borderColor: GradientColor.BLUE[0],
```

Reply:
Linter complains "no magic numbers" in this case, shoud I comment linter rule above?

Reply:
OK, leave the `colorIndex` then.